[DOCS] Added descriptions to Camera2D

sednave opened the pull request:
Added descriptions to some member variables.

Commit:
Added descriptions to some members of Camera2D

cbscribe commented on doc/classes/Camera2D.xml:
```diff
@@ -333,6 +333,7 @@
 			Bottom margin needed to drag the camera. A value of [code]1[/code] makes the camera move only when reaching the edge of the screen.
 		</member>
 		<member name="drag_margin_h_enabled" type="bool" setter="set_h_drag_enabled" getter="is_h_drag_enabled">
+			If [code]true[/code] this camera only moves when reaching the horizontal drag margins. If [code]false[/code] the camera moves horizontally regardless of margins.
```
Please add default value

cbscribe commented on doc/classes/Camera2D.xml:
```diff
@@ -361,6 +363,7 @@
 			Right scroll limit in pixels. The camera stops moving when reaching this value.
 		</member>
 		<member name="limit_smoothed" type="bool" setter="set_limit_smoothing_enabled" getter="is_limit_smoothing_enabled">
+			If [code]true[/code] smooth camera when reaching camera limits.
```
This feels incomplete. I think it needs an explanation of what "smooth camera" means.

cbscribe commented on doc/classes/Camera2D.xml:
```diff
@@ -371,8 +374,10 @@
 		<member name="rotating" type="bool" setter="set_rotating" getter="is_rotating">
 		</member>
 		<member name="smoothing_enabled" type="bool" setter="set_enable_follow_smoothing" getter="is_follow_smoothing_enabled">
+			If [code]true[/code] this camera moves with smoothing.
```
As above, describe what smoothing does.

cbscribe commented on doc/classes/Camera2D.xml:
```diff
@@ -371,8 +374,10 @@
+			If [code]true[/code] this camera moves with smoothing.
 		</member>
 		<member name="smoothing_speed" type="float" setter="set_follow_smoothing" getter="get_follow_smoothing">
+			Speed at which the camera moves to its current smoothing destination.
```
The camera's speed when moving to its...

Commit:
Updated Camera2D docs with suggestions

Removed the description for limit_smoothed.
Added a new description for rotating.
Added default values for the booleans.

cbscribe commented on doc/classes/Camera2D.xml:
```diff
@@ -369,10 +371,13 @@
+			If [code]true[/code] the camera can rotate during gameplay. Default value: [code]false[/code].
 		</member>
 		<member name="smoothing_enabled" type="bool" setter="set_enable_follow_smoothing" getter="is_follow_smoothing_enabled">
+			If [code]true[/code] the camera gradually changes its velocity vector. Default value: [code]false[/code].
```
Hmm, this makes it sound like the camera just moves by itself.  How about:

    If [code]true[/code] the camera's velocity changes gradually, preventing sudden starts and stops.

cbscribe commented on doc/classes/Camera2D.xml:
```diff
@@ -369,10 +371,13 @@
+			If [code]true[/code] the camera gradually changes its velocity vector. Default value: [code]false[/code].
 		</member>
 		<member name="smoothing_speed" type="float" setter="set_follow_smoothing" getter="get_follow_smoothing">
+			The camera's acceleration if smoothing is enabled.
```
Not sure about this one. Is it an acceleration? The name says speed.

cbscribe commented on doc/classes/Camera2D.xml:
```diff
@@ -333,6 +333,7 @@
 			Bottom margin needed to drag the camera. A value of [code]1[/code] makes the camera move only when reaching the edge of the screen.
 		</member>
 		<member name="drag_margin_h_enabled" type="bool" setter="set_h_drag_enabled" getter="is_h_drag_enabled">
+			If [code]true[/code] this camera only moves when reaching the horizontal drag margins. If [code]false[/code] the camera moves horizontally regardless of margins. Default value: [code]true[/code].
```
Everywhere else you say "the camera" instead of "this camera". You should make it consistent.

Commit:
Updated documentation

Changed all instances of "this camera" to "the camera" for consistency.
Changed descriptions for smoothing_enabled and smoothing_speed.

cbscribe commented:
Looks good, please squash the commits.

akien-mga commented:
Squashed and merged manually as 8f382879c.
